examples/apm-sample/app.js

```diff
@@ -0,0 +1,25 @@
+var config = require("config");
+
+if (config.Raygun.Key === "YOUR_API_KEY") {
```

Comment:
Should we be encouraging no API keys in code to set a good standard? Could be via .env

Reply:
Because this is an example app, I would think this is not necessary to add in

Reply:
`raygun-apm` gets the api key from env variable, which we should probably do as well eventually, out of scope for this release IMO.